app/components/Button/actions/login.ts

```
@@ -0,0 +1,37 @@
+"use server";
+import { VerifyLoginPayloadParams } from "thirdweb/auth";
+import { thirdwebAuth } from "@app/lib/sdk/thirdweb/auth";
+import { cookies } from "next/headers";;
+
+export const generatePayload = thirdwebAuth.generatePayload;
+
+export async function login(payload: VerifyLoginPayloadParams): Promise<{ valid: Boolean }> {
```

Biome

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)